src/kokkos/test/kokkos/OneHistoContainer_t.cc

```diff
@@ -0,0 +1,143 @@
+#include "KokkosCore/kokkosConfigCommon.h"
+#include "KokkosCore/kokkosConfig.h"
+#include <algorithm>
+#include <cassert>
+#include <iostream>
+#include <random>
+#include <limits>
+
+#include "KokkosCore/HistoContainer.h"
+using team_policy = Kokkos::TeamPolicy<KokkosExecSpace>;
+using member_type = Kokkos::TeamPolicy<KokkosExecSpace>::member_type;
+
+template <typename T, int NBINS = 128, int S = 8 * sizeof(T), int DELTA = 1000>
+void go() {
+  std::mt19937 eng;
+
+  int rmin = std::numeric_limits<T>::min();
+  int rmax = std::numeric_limits<T>::max();
+  if (NBINS != 128) {
+    rmin = 0;
+    rmax = NBINS * 2 - 1;
+  }
+
+  std::default_random_engine generator (1234);
+  std::uniform_int_distribution<T> rgen(rmin, rmax);
+
+  constexpr uint32_t N = 12000;
+  Kokkos::View<T*,KokkosExecSpace> v_d("v_d",N);
+  typename Kokkos::View<T*,KokkosExecSpace>::HostMirror v_h("v_h",N);
+
+  using Hist = HistoContainer<T, NBINS, N, S>;
+  std::cout << "HistoContainer " << Hist::nbits() << ' ' << Hist::nbins() << ' ' << Hist::capacity() << ' '
+            << (rmax - rmin) / Hist::nbins() << std::endl;
+  std::cout << "bins " << int(Hist::bin(0)) << ' ' << int(Hist::bin(rmin)) << ' ' << int(Hist::bin(rmax)) << std::endl;
+
+  for (int it = 0; it < 5; ++it) {
+    for (long long j = 0; j < N; j++)
+      v_h(j) = rgen(eng);
+    if (it == 2)
+      for (long long j = N / 2; j < N / 2 + N / 4; j++)
+        v_h(j) = 4;
+
+    Kokkos::deep_copy(v_d,v_h);
+    
+    using TeamHist = HistoContainer<T, NBINS, N, S, uint16_t>;
+
+    Kokkos::View<TeamHist*,KokkosExecSpace> histo_d("histo_d",1);
+    auto histo_h = Kokkos::create_mirror_view(histo_d);
+
+    Kokkos::parallel_for("set_zero",Kokkos::RangePolicy<KokkosExecSpace>(0,TeamHist::totbins()),
```

Comment:
Please pass the execution space object, e.g.
```suggestion
    Kokkos::parallel_for("set_zero",Kokkos::RangePolicy<KokkosExecSpace>(KokkosExecSpace(), 0,TeamHist::totbins()),
```
so that the serial build of the unit test can be ran on a machine without a GPU.